app/helpers/court_dates_helper.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Helper methods for court_dates
+module CourtDatesHelper
+  def when_do_we_have_court_dates(casa_case)
+    date_query = casa_case.court_dates.includes(:hearing_type).ordered_ascending
+    last_court_date = date_query.last
+    first_court_date = date_query.first
+
+    if last_court_date.date < Date.current
+      "past"
+    elsif last_court_date.date > Date.current && first_court_date.date > Date.current
```

Comment:
**suggestion:** it looks to me that you could get away with only `first_court_date.date > Date.current` instead of `last_court_date.date > Date.current && first_court_date.date > Date.current`. WDYT?

Reply:
derp! agree! 🙂